src/site/content/en/blog/index.njk

```diff
@@ -1,14 +1,14 @@
 ---
 layout: layout
```

Comment:
I had a couple ideas about this file. If we want to use it as a general purpose pagination page then maybe we should rename the template to `paged.njk` or something like that. Separately we would still have `blog.njk` but maybe it just extends `paged.njk` to overwrite some of its yaml?

Reply:
Any thoughts on the `paged.njk` idea?